frontend/src/app/modules/home/projects/edit/project-alert/project-alert.component.html

```diff
@@ -0,0 +1,59 @@
+<p-panel header="Alert" class="panel">
+    <form [formGroup]="editForm" class="item">
+        <label class="item-definition">
+            <div>Alert Category</div>
+            <div class="item-sub-text">The category of alerts, depending on the parameter, will work differently</div>
+        </label>
+        <div class="item-field">
+            <div class="item-form-input p-input-icon-right">
+                <p-dropdown [options]="alertCategoryValue" optionLabel="label"
+                    formControlName="alertCategory" optionValue="value" [style]="{'width':'100%'}"></p-dropdown>
+            </div>
+        </div>
+    </form>
+    <form [formGroup]="editForm" *ngIf="this.editForm.controls.alertCategory.value=== 3"
+         class="item">
+        <label class="item-definition-special">
+            <div>Alert Count</div>
+            <div class="item-sub-text">How many alerts will you receive</div>
+        </label>
+        <div class="item-field">
+            <div class="item-form-input p-input-icon-right">
+                <input formControlName="alertsCount" pInputText placeholder="Alerts count" class="item-form-field">
+                <div *ngIf="alertsCount.touched && alertsCount.invalid" class="p-error">
+                    <small *ngIf="alertsCount.errors.required">Сan not be empty</small>
+                </div>
+            </div>
+        </div>
+    </form>
+    <form [formGroup]="editForm" *ngIf="this.editForm.controls.alertCategory.value=== 3" class="item">
```

Comment:
What is 3? Please no magic numbers. Compare with Enum type.

Reply:
Fix